hw/ip/otp_ctrl/data/BUILD

```diff
@@ -474,15 +474,16 @@ otp_image(
     for i in range(0, 7)
 ]
 
-# Represents a DEV state OTP image emulating the state of the device after the
-# exit test token has been applied and before running individualization.
-# The following partitions are missing to ensure the image is initialize with
-# default values: SECRET1, SECRET2, HW_CFG.
-# The following partitions are expected to be configured in previous lifecycle
-# stages: SECRET0, CREATOR_SW, OWNER_SW.
+# Represents a TEST_UNLOCKED1 state OTP image emulating the state of the device
+# after the test tokens have been applied and before running individualization.
+# The following partitions are missing to ensure the image is initialized with
+# values that would be present prior to the final individualization
+# manufacturing stage: SECRET1, SECRET2, HW_CFG.
+# The following partitions are expected to be configured in previous
+# manufacturing stages: SECRET0, CREATOR_SW, OWNER_SW.
 otp_image(
-    name = "img_dev_initial",
-    src = ":otp_json_dev",
+    name = "img_test_unlocked1_initial",
```

Comment:
We should document a table of OTP images once this is more stable to help others understand the start/end states of each step of the provisioning process.